### DISCUSSION TITLE
posts/2023/06/miss-what/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#6] [General]
# posts/2023/06/miss-what/

如何停止对你的念想？

https://guozheng.netlify.app/posts/2023/06/miss-what/

**Comment by CyrusYip** [comment]
> 她给了我们每人一瓶藿香正气液，就像她会喜欢取纸质高铁票这样的仪式性小举动

原来还能取纸质高铁票啊，我以为取消了。我今天也取了，看纸质票比在看手机方便。

> 下船之前，她给我剥了三颗荔枝。

隔着屏幕都觉得很甜。

**Reply by residualsun1** [reply]
纸质确实要比手机方便。如果不是跟着她俩一起去旅行，我估计永远都不知道还可以取高铁票这回事😂，即使知道可能也不会花心思去做。话说回来，又想起和我初中玩伴去旅行的时候，他就说会把在商店买东西得到的小票当作书签，就像《花束般的恋爱》中的男女主会拿电影票存根当书签。感觉和她俩相比，我简直已经是一个“无聊的大人”。不过还为时不晚，毕竟，“人生的快乐秘笈，是不要忘记童心”——《银魂》

**Reply by CyrusYip** [reply]
> 如果不是跟着她一起去旅行，我估计永远都不知道还可以取高铁票这回事😂，即使知道可能也不会花心思去做。

我六七年前坐高铁还是用高铁票入站，出站时高铁票被机器吸进去之后从上面弹出来，感觉特别神奇。那时的票和身份证差不多大，蓝色横向的。后面强制用身份证入站后就没怎么拿过票了。

我昨天拿那个是行程信息，白色纵向的，像商场的小票，好丑啊。你拿到高铁票是哪种？

**Reply by residualsun1** [reply]
我拿的好像就是你所说的六七年前那种，长这样：

![tickect](https://cdn.jsdelivr.net/gh/residualsun1/blog-static/images/2023/06/06-03-ticket.jpg)

感觉还是有收藏价值的。

**Reply by residualsun1** [reply]
> 隔着屏幕都觉得很甜。

有时候我会天真地想，生活中短暂出现的美好被记录以后，它可能会在另一个时空成为现实。

**Reply by CyrusYip** [reply]
我终于知道怎么拿蓝色的票了，原来它现在叫报销凭证。我之前取那个超市小票那样的叫行程信息。

**Reply by residualsun1** [reply]
哈哈，我猜你是在高铁站的自助机器那里拿的。

我四月底公费去广西参加了一个社区保护地的会议，但是出发时忘记取票报销了，回来的时候还可以取，就是在高铁站的自助机器拿的。我想现在用票的人不多了，毕竟在网络上买票后直接刷身份证就行，一般要票的话应该是用来报销，可能因为这样才叫”报销凭证“吧。

没想到你还会记得高铁小票【我有点恍惚感 : )

**Reply by CyrusYip** [reply]
确实是在自助机器拿的，去服务台拿也一样，要报销凭证才是蓝色硬质卡片。我去服务台要高铁票的时候，工作人员很诧异，说可以用手机看。我说我爱看纸质的。另一个工作人员听到还冲我竖起大拇指。

很多人说手机很方便，可以做很多事。我最近感觉以前的一些方式反而方便。

比如用纸钞给钱比手机扫码快。手机扫码过程很多，首先要解锁手机、打开微信/支付宝、打开扫码支付，如果是自己主动扫码就还要输入金额和密码。

还有些餐厅，明明店员不少，但客人只能用小程序下单。小程序也是个门槛，首先你得有智能手机和微信/支付宝，对于一些不懂的人来说会很无奈。我就喜欢看实体菜单，眼睛扫过去很快就看完了，手机还得慢慢翻。

> 没想到你还会记得高铁小票【我有点恍惚感 : )

哈哈，想不到我会继续一年前的讨论吗？

**Reply by residualsun1** [reply]
> 我去服务台要高铁票的时候，工作人员很诧异，说可以用手机看。我说我爱看纸质的。另一个工作人员听到还冲我竖起大拇指。

清流😁。

> 比如用纸钞给钱比手机扫码快。手机扫码过程很多，首先要解锁手机、打开微信/支付宝、打开扫码支付，如果是自己主动扫码就还要输入金额和密码。

这个角度还挺清奇的，我已经对现金支付几乎没有概念和想象了，好像很久以前的相关感受都不见了，完全习惯微信支付让我觉得这个就是从一而终的支付方式，我很少想过它是否麻烦。可能这要和现金支付的感受对比才明显吧。

> 还有些餐厅，明明店员不少，但客人只能用小程序下单。小程序也是个门槛，首先你得有智能手机和微信/支付宝，对于一些不懂的人来说会很无奈。我就喜欢看实体菜单，眼睛扫过去很快就看完了，手机还得慢慢翻。

说到这个的话，我第一感觉是麻烦，有时候开了小程序真是眼花缭乱，各种广告、优惠券满天飞，时而又弹出一些消息通知的权限申请（比如优惠券到账啥的），我很不喜欢。每次我和朋友出去吃饭，如果有小程序点单我都会“摆烂”，让他们来点菜，因为他们会弄团购，有的还要再到美团、抖音去整——这些已经麻烦到让我丝毫不想动弹，不如直接看菜单点菜。

> 哈哈，想不到我会继续一年前的讨论吗？

对呀，看到你说到一年前的事情，我会在一个很短的时间有一小段恍惚感，给你分享一张我在豆瓣上看的meme图。

![](https://cdn.jsdelivr.net/gh/residualsun1/blog-static/images/2024/10/10-02-memory.jpg)

**Reply by CyrusYip** [reply]
哈哈，在用小程序这件事上咱们是一样的，我也是讨厌小程序收集个人信息，去吃饭让朋友来弄。

看了你这个图，十年前的我肯定想象不到我现在的情况。
